projects/schematics/src/migrations/3_0/constructor-deprecations/data/user-register.effect.migration.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  SPARTACUS_CORE,
+  NGRX_STORE,
+  AUTH_SERVICE,
+  ACTIONS,
+  USER_CONNECTOR,
+  USER_REGISTER_EFFECT,
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const USER_REGISTRATION_EFFECT_MIGRATION: ConstructorDeprecation = {
+  // projects\core\src\user\store\effects\user-register.effect.ts
+  class: USER_REGISTER_EFFECT,
+  importPath: SPARTACUS_CORE,
+  deprecatedParams: [
+    {
+      className: ACTIONS,
+      importPath: NGRX_STORE,
+    },
+    {
+      className: USER_CONNECTOR,
+      importPath: SPARTACUS_CORE,
+    },
+  ],
+  removeParams: [],
```

Comment:
no need since nothing is removed and its an optional param

Reply:
Done